services/scraper/scraper.go

```diff
@@ -175,7 +177,7 @@ func getResultFromContext(context *colly.Context) *models.Result {
 
 	result, err := models.GetResultById(resultID)
 	if err != nil {
-		logs.Error("Failed to get result by ID", err.Error())
+		logs.Error("Failed to get result by ID", resultID, err.Error())
```

Comment:
According to @junan's comment. How about
```suggestion
		logs.Error("Failed to get result by ID: ", resultID, err.Error())
```

Reply:
added in [d37f39c](https://github.com/carryall/go-google-scraper-challenge/pull/65/commits/d37f39c9f8b9ba158492c13e6feaf4865b7b8d0a)